.github/workflows/test.yml

```diff
@@ -31,3 +31,42 @@ jobs:
         run: yarn build
       - name: Run unit tests
         run: yarn test
+  build:
```

Comment:
So is this still necessary? Or is the push step necessary?

Reply:
I think it is. It will try to build the Dockerfile on all branches except `master`. That way we will know if something is wrong in build phase before actually trying to publish the image.

Reply:
Sure, that makes sense. But we can drop push step, right?

Reply:
Oh, it's building as well, nvm